### ISSUE TITLE
Windows Error 0xc000001d

### ISSUE THREAD

**Issue by Ruubb** [qilingframework/qiling#1337]
***Describe the bug**
Hi, I am running into a problem when I try to 'inspect' my own hello_world.exe. The hello world code looks like this:
```
#include <iostream>

int main()
{
    std::cout << "Hello World!\n";
}
```

This is my python code for using Qiling

**Sample Code**
```python
#!/usr/bin/env python3
# 
# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
#

import platform, sys, unittest
from typing import List

from unicorn import UcError

sys.path.append("..")
from qiling import Qiling
from qiling.const import QL_STOP, QL_VERBOSE
from qiling.os.const import POINTER, DWORD, HANDLE
from qiling.exception import QlErrorSyscallError
from qiling.os.windows import utils
from qiling.os.windows.wdk_const import *
from qiling.os.windows.api import *
from qiling.os.windows.fncc import *
from qiling.os.windows.dlls.kernel32.fileapi import _CreateFile

from qiling import Qiling

if __name__ == "__main__":
    #ql = Qiling(["c:/examples/rdriver.sys"], "c:/examples/rootfs/x8664_windows",  stop=QL_STOP.STACK_POINTER, libcache=True)
    ql = Qiling(["c:/examples/hello world.exe"], "c:/examples/rootfs/x8664_windows", archtype='x8664', ostype='Windows')

    ql.run()
```

I would expect that to work fine, but this is the output I am receiving:

```
PS C:\Users\User>  C:/Users/User/AppData/Local/Programs/Python/Python311/python.exe c:/examples/test.py
[=]     Initiate stack address at 0x7ffffffde000 
[=]     Loading c:/examples/hello world.exe to 0x140000000
[=]     PE entry point at 0x1400112c6
[=]     TEB is at 0x6000000
[=]     PEB is at 0x60001f0
[=]     LDR is at 0x60009c0
[=]     Loading ntdll.dll ...
[=]     Done loading ntdll.dll
[=]     Loading kernel32.dll ...
[=]     Loading kernelbase.dll ...
[=]     Done loading kernelbase.dll
[=]     Done loading kernel32.dll
[=]     Loading mscoree.dll ...
[=]     Calling mscoree.dll DllMain at 0x1806bd3b0
Traceback (most recent call last):
  File "c:\examples\test.py", line 26, in <module>
    ql = Qiling(["c:/examples/hello world.exe"], "c:/examples/rootfs/x8664_windows", archtype='x8664', ostype='Windows')
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\qiling\core.py", line 184, in __init__
    self.loader.run()
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\qiling\loader\pe.py", line 716, in run
    self.load(pe)
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\qiling\loader\pe.py", line 795, in load
    super().load_dll(each, self.is_driver)
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\qiling\loader\pe.py", line 255, in load_dll
    self.call_dll_entrypoint(dll, dll_base, dll_len, dll_name)
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\qiling\loader\pe.py", line 305, in call_dll_entrypoint
    self.ql.emu_start(entry_point, exit_point)
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\qiling\core.py", line 753, in emu_start
    self.uc.emu_start(begin, end, timeout, count)
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\unicorn\unicorn.py", line 545, in emu_start
    status = _uc.uc_emu_start(self._uch, begin, until, timeout, count)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [WinError -1073741795] Windows Error 0xc000001d
```

I tried a hello_world driver as well (since my goal was to debug a driver), but that gave me the same error. I would expect something is wrong with my dlls/registries (The supplied script did not copy any of the registries, so I exported them by hand). 

**Expected behavior**
A working example

**Additional context**
I was wondering if this on me, or that something else is wrong. So, how would I debug if I am doing something wrong? I tried copy-pasting my x64 dll from windows to the rootffs again, but without succes.
[hello world.zip](https://github.com/qilingframework/qiling/files/11145432/hello.world.zip)

**Comment by elicn** [comment]
Unicorn installation issue.

**Comment by elicn** [comment]
Unicorn installation issue.

**Comment by not-inc-neg** [comment]
Posting in case anyone from the future faces this issue. Spent the last day or so grappling with this exact error (mscoree.dll, 0xc000001d), having followed the Qiling "Pyenv Installation with latest dev branch (recommended)" instructions.

My environment:

- Virtualbox 7.0.14 r161095
- Windows 10 x64 Host
- Windows 10 x64 Guest
- Guest has Python 3.11.8 x64 installed
- AMD Ryzen 5800X3D CPU

tl;dr, Hyper-V running on the Windows _host_, causes performance issues and limits **CPU features** available in the guest VM (if you see the green turtle in bottom right of the guest Window, this could be the root cause for your 0xc000001d error). Disable HyperV on the host and try again. See post #2 for guidance: https://forums.virtualbox.org/viewtopic.php?f=1&t=62339

Source of "illegal instruction" error is python3.exe issuing VZEROUPPER instruction, which is an AVX instruction that Virtualbox may not "pass through" to the guest VM because Hyper-V is running on the host. You can confirm if this is the case by running HWINFO64 (Summary only) on the guest - confirm if AVX/AVX2 is green - if not, python3.exe will crash with the above error).

Once I disabled HyperV on the host, AVX/2 instructions became available in the guest and Qiling now runs my "HelloWorld.exe" without zero issues. Happy Days.
